src/net.h

```diff
@@ -824,7 +825,7 @@ class CConnman
         m_max_outbound_block_relay = std::min(MAX_BLOCK_RELAY_ONLY_CONNECTIONS, m_max_automatic_connections - m_max_outbound_full_relay);
         m_max_automatic_outbound = m_max_outbound_full_relay + m_max_outbound_block_relay + m_max_feeler;
         m_max_inbound = std::max(0, m_max_automatic_connections - m_max_automatic_outbound);
-        m_max_inbound_full_relay = std::max(0, static_cast<int>(FULL_RELAY_INBOUND_PCT * m_max_inbound));
+        m_max_inbound_full_relay = std::min(1, std::max(0, static_cast<int>(connOptions.m_full_relay_inbound_ratio * m_max_inbound)));
```

Comment:
the added `min(1, ...)` seems incorrect (and breaks the test). It means we can't ever have more than one FR inbound?